atmo/templates/atmo/spark-job-detail.html

```diff
@@ -21,14 +23,99 @@
 </div>
 <div class="row">
   <div class="col-sm-9">
-    <div id="notebook-content"
+    <ul class="nav nav-tabs" role="tablist">
+      <li role="presentation" class="active">
+        <a href="#notebook" aria-controls="notebook" role="tab" data-toggle="tab">Notebook</a>
+      </li>
+      <li role="presentation">
+        <a href="#results" aria-controls="results" role="tab" data-toggle="tab">Results</a>
+      </li>
+    </ul>
+    <div class="tab-content">
+      <div role="tabpanel" class="tab-pane active" id="notebook">
+        <div id="notebook-content"
         data-download-url="{% url 'jobs-download' id=job.id %}"
         data-content-given="{% if notebook_content %}true{% else %}false{% endif %}">
-      {% if notebook_content %}
-      <textarea class="hidden">{{ notebook_content }}</textarea>
-      {% else %}
-      Loading notebook content&hellip;
-      {% endif %}
+        {% if notebook_content %}
+        <textarea class="hidden">{{ notebook_content }}</textarea>
+        {% else %}
+        Loading notebook content&hellip;
+        {% endif %}
+        </div>
+      </div>
+      <div role="tabpanel" class="tab-pane" id="results">
+        {% with results=job.get_results %}
+        <div class="panel panel-default">
+          <div class="panel-heading">Artifacts</div>
+          <div class="panel-body">
+            <p>
+            {% if job.is_public %}
+            The artifacts of this job are public and can be downloaded using the links below.
+            {% else %}
+            The artifacts of this job are not publicly available. AWS credentials are required
+            in order to access them, either via the <a href="https://console.aws.amazon.com/">AWS Console</a>
+            or using the <a href="https://aws.amazon.com/cli/">AWS CLI</a>. Only the latest version of each artifact is kept,
+            so the next job run will likely override these files.
+            </p>
+            <p>
+            You can download the files to your machine using the AWS CLI:
+            <pre>aws s3 cp &lt;s3 path&gt; .</pre>
+            {% endif %}
+            </p>
+          </div>
+          <ul class="list-group">
+          {% for item in results.data %}
+            <li class="list-group-item">
+              {% if job.is_public %}
+                {% if item|is_notebook %}
+                  <a href="{{ settings.PUBLIC_NB_URL }}{{ item }}">{{ item }}</a>
```

Comment:
Hm, I don't think we should require nbviewer outside of our control, since it adds a dependency to the outside. Could we instead fetch the notebook from S3 and display it with notebook.js like we do with the current job's notebook?